src/main/java/com/codenear/butterfly/support/domain/Inquiry.java

```diff
@@ -28,4 +44,16 @@ public class Inquiry extends BaseEntity {
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "member_id", nullable = false)
     private Member member;
+
+    public void toggleStatus() {
+        if (this.status == PENDING)
+            this.status = ANSWERED;
+
+        if (this.status == ANSWERED)
+            this.status = PENDING;
```

Comment:
이 부분
```suggestion
        if (this.status.equals(PENDING))
            this.status = ANSWERED;

        if (this.status.equals(ANSWERED))
            this.status = PENDING;
```
이런식으로 수정하는거 어때?